Sources_v3/Workers/Background/NewChannelObserver.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+final class NewChannelObserver<ExtraData: ExtraDataTypes>: Worker, NSFetchedResultsControllerDelegate {
+    private let channelListQueryUpdated: ChannelListQueryUpdater<ExtraData>
+    private let fetchRequest = ChannelDTO.channelListFetchRequestAll
+    
+    private(set) lazy var frc: NSFetchedResultsController<ChannelDTO> = .init(fetchRequest: self.fetchRequest,
+                                                                              managedObjectContext: self.database.writableContext,
+                                                                              sectionNameKeyPath: nil,
+                                                                              cacheName: nil)
```

Comment:
You can use `ListDatabaseObserver`. See `MessageSender` for reference

Reply:
I started with `ListDatabaseObserver` but I don't need `ItemCreator` which is mandatory, so I don't think I should use some class that has functionality which is not needed and doesn't bring much simplicity in my case.

Reply:
I see, however, I'd suggest using the `ListDatabaseObserver` anyway. It will enable us to do some further optimizations, like sharing the underlying FRCs for multiple observers, etc., without having to touch the "public" api.

Maybe, we could introduce some convenience init for the observers without the creator if `Item == DTO`.